subprojects/packagefiles/openblas/benchmark/pybench/meson.build

```diff
@@ -0,0 +1,48 @@
+#
+# Taken from SciPy (of course)
+#
+project(
+  'openblas-wrap',
+  'c', 'fortran',
+  version: '0.1',
```

Comment:
This should probably be 0.3.28.

Reply:
Actually the benchmark directory should probably not even be part of this submission, removing :)